versions.txt

```diff
@@ -1,8 +1,8 @@
 # Represents the latest stable release of the CloudWatch Agent
-cloudwatch-agent=1.300034.0b498
+cloudwatch-agent=1.300035.0b545
 
 # Represents the current release of the CloudWatch Agent Operator.
-operator=1.1.0
+operator=1.2.0
```

Comment:
Let's keep these versions as is. We will update the versions.txt and the images in the helm chart once we start an eks add-on deployment

Reply:
reverted operator version in `versions.txt`